pkg/ee/conversion/helm.go

```diff
@@ -307,6 +308,10 @@ func ConvertDatacenters(datacenterMeta map[string]provider.DatacenterMeta, globa
 		seed.Kind = "Seed"
 		seed.Namespace = targetNamespace
 
+		if pauseProvisioning {
+			seed.Annotations[common.SkipReconcilingAnnotation] = ""
```

Comment:
Add nil check to avoid panics e.g.
```
                       if seed.Annotations == nil {
                               seed.Annotations = make(map[string]string)
                       }
```

Reply:
Good catch, fixed.